client/signup/jetpack-connect/plans.jsx

```diff
@@ -12,7 +12,7 @@ import { bindActionCreators } from 'redux';
 import { getPlansBySite } from 'state/sites/plans/selectors';
 import { getFlowType } from 'state/jetpack-connect/selectors';
 import Main from 'components/main';
-import ConnectHeader from './connect-header';
+import StepHeader from '../step-header';
```

Comment:
AFAIK no. Even if they were, it would be problematic when you import multiple members from the same file.